worlds/mlss/__init__.py

```diff
@@ -55,29 +55,28 @@ class MLSSWorld(World):
     settings: typing.ClassVar[MLSSSettings]
     item_name_to_id = {name: data.code for name, data in item_table.items()}
     location_name_to_id = {loc_data.name: loc_data.id for loc_data in all_locations}
-    required_client_version = (0, 4, 5)
+    required_client_version = (0, 5, 0)
 
-    disabled_locations: List[str]
+    disabled_locations: Set[str] = set()
```

Comment:
```suggestion
    disabled_locations: Set[str]
```

Don't initialize here, that will make a single set shared across all slots. Only declare